docs/faqs.md

```diff
@@ -409,3 +409,9 @@ merging to the master branch. Feel free to contribute to a new Github Action tha
 
 No. The [pylint](v2_linter.md) has an important role of keeping any recipe prepared for [Conan v2 migration](v2_migration.md). In case you are having
 difficult to understand [linter errors](linters.md), please comment on your pull request about the problem to receive help from the community.
+
+## How long can I be inactive before being removed from the authorized users list?
+
+For now, it's configured for 4 months. It's computed based on your last commit, not comments or issues.
```

Comment:
I think this is way too short. we will lose some contributors with this change. Maybe something like 12 months could be better? (all in all, I don't understand what cost you are trying to mitigate)

Reply:
@uilianries can we increase to some reasonable amount like 44 months? 4 months is way too short, basically, e.g. parental leave lasts longer.

Reply:
`It's computed based on your last commit`

this is also not safe, as an arbitrary commit date could be set on `git commit`

Reply:
Security concerns 😟

Reply:
What about 6 or 8 months https://github.com/conan-io/conan-center-index/commits?author=grafikrobot looking like worst was 7 months 
![Screenshot_20221027-212802_Chrome](https://user-images.githubusercontent.com/16867443/198502619-45611480-789f-4f5a-9a59-9e8963fb3c59.jpg)

I think preventing a robot uprising is a good bench mark 😉 but I think keeping it as tight as possible is going to be where we land

Reply:
> this is also not safe, as an arbitrary commit date could be set

The merge commits are managed by us so I think it's safe to use them in this case

Reply:
I understand that you need to protect the infrastructure, but I don't think that it's a good way to make it at the expense of normal contributors. The system already feels quite hostile to contributors, with opaque CI failures for several days. These failures also prevent new users from being authorized (there was no change in https://github.com/conan-io/conan-center-index/commits/master/.c3i/authorized_users.yml for 2weeks between the October 3rd and 17th)
If the purpose of the authorization is to prevent bot attacks, then it should check only the author of the pull request, but it should not check the commit themselves. There are quite often users which have different commit identities, and are asked to rewrite the history of a branch in order to satisfy the authorization.

Reply:
@ericLemanissier This is the first step, and can be improved, more points can added to be considered as activity like comments and issues. Same for those users which use different accounts.

Reply:
> What about 6 or 8 months

8 it's a good number IHMO.

Reply:
Yes, a larger number would be better. I think at least a year would be better. It's not uncommon for me to pick a software project up after a year of not touching it and finish it up. If someone gets access and works for several months on getting a single package to work, it'd be annoying to have to reapply just because nothing had been merged in.

Reply:
I bumped this to the team so we can have a meeting to change it :) thanks for all the suggestions